pkg/operator/operator.go

```diff
@@ -160,6 +161,16 @@ func Start(ctx context.Context, buildInfo starboard.BuildInfo, operatorConfig et
 		}).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to setup vulnerabilityreport reconciler: %w", err)
 		}
+
+		if operatorConfig.VulnerabilityScannerReportTTL != time.Duration(0) {
```

Comment:
Is it worth adding a function to perform the `operatorConfig.VulnerabilityScannerReportTTL != time.Duration(0) `? We perform it in two files and import time in multiple locations.

We could add this function in to: `pkg/operator/etc/config.go`. Then it would also be easy to write a small test for the function.